packages/headless/src/modules/store/services/index.ts

```diff
@@ -98,4 +99,37 @@ export class StoreService {
       );
     return this.helper.serviceResponse.successResponse(store, HttpStatus.OK);
   }
+
+  async getAllStores(condition: GetAllStoresQuery): Promise<GetStoreResponse> {
+    const { adminEmail, skip, limit } = condition;
+    let stores: any = [];
+    const query: Record<string, any> = this.generateSearchQuery(condition);
+    stores = await this.storeRepo.getAllStores(
+      { ...query, email: adminEmail },
```

Comment:
email could be added to the `query`. just needed to update the `generateSearchQuery` function